v2/netconf/ops/model.go

```diff
@@ -0,0 +1,77 @@
+package ops
+
+import "encoding/xml"
+
+const (
+	// Configuration Datastores
```

Comment:
there are additional datastores.  https://tools.ietf.org/html/rfc8342

Reply:
Added intended and operational, per rfc.